_maps/map_files/stations/deltastation.dmm

```diff
@@ -27970,6 +27966,10 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
+/obj/structure/disposalpipe/segment{
+	dir = 4;
+	icon_state = "pipe-c"
+	},
```

Comment:
You've still got a var edited corner pipe

Reply:
fixed